vladyslav.taran/Homework4/task_2/task_2.rb

```diff
@@ -0,0 +1,16 @@
+def space_size(str)
```

Comment:
and why do you need this method?

Reply:
I understood it works without this method.

Reply:
I understood it works without this method.